data/movements/movements.xml

```diff
@@ -29,8 +29,7 @@
 	<movevent event="StepIn" itemid="1387" script="citizen.lua" />
 
 	<!-- Walkback when walking on quest chest -->
-	<movevent event="StepIn" itemid="1738" script="walkback.lua" />
-	<movevent event="StepIn" itemid="1740" script="walkback.lua" />
+	<movevent event="StepIn" itemid="1738;1740" script="walkback.lua" />
```

Comment:
Missed it before - that's also the kind of datapack change I wanted to avoid. I assume you've tested using that one.

Reply:
Of course :) It was just proof of concept.